CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Refactored NavBarItem to use IconButton directly
 - Change control bar theme opacity to 1
 - [Chat Demo] Rename misspelled file names and imports
+- Set amazon-chime-sdk-js in package.json dependencies
```

Comment:
I think we'll need to update the docs to call this change out

- [In the introduction](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/components/introduction.stories.mdx#getting-started)
- [In the migration guide](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/src/components/migrationToV2.stories.mdx#important---version-2-of-the-library-also-requires-a-dependency-to-version-24-or-higher-of-the-amazon-chime-sdk-js)

Reply:
In the migration guide, could we add a note about builders having to uninstall amazon-chime-sdk-js from their direct deps?

Reply:
Updated both documents. I do not think we need to uninstall over version of amazon-chime-sdk-js